File: sample_java/src/main/java/com/memtrip/sample_java/AppNotificationChannel.java
package com.memtrip.sample_java;

import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.content.Context;
import android.os.Build;

import com.memtrip.exoeasy.sample_java.R;

class AppNotificationChannel {

    private final Context context;
    private final NotificationManager notificationManager;

    AppNotificationChannel(Context context) {
        this.context = context;
        notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
    }

    void create() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            notificationManager.createNotificationChannel(new NotificationChannel(
                    context.getString(R.string.app_notification_channel_id),
                    context.getString(R.string.app_notification_channel_name),
                    NotificationManager.IMPORTANCE_LOW));
        }
    }
}

File: sample/src/main/java/com/memtrip/exoeasy/sample/AppNotificationChannel.kt
package com.memtrip.exoeasy.sample

import android.app.NotificationManager
import android.app.NotificationChannel
import android.content.Context
import android.os.Build

class AppNotificationChannel(
    private val context: Context,
    private val notificationManager: NotificationManager =
        context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
) {

    fun create() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            notificationManager.createNotificationChannel(NotificationChannel(
                context.getString(R.string.app_notification_channel_id),
                context.getString(R.string.app_notification_channel_name),
                NotificationManager.IMPORTANCE_LOW))
        }
    }
}
